docs/badges/distribution.rst

```diff
@@ -0,0 +1,311 @@
+Distribution
+============
+
+    System allows already earned badges distribution to external services via backends.
+
+Badge distribution backends are Credentials plugins (installable applications) which highly rely on internal `badges` application, but extend its functions.
+
+Each backend may be responsible for:
+
+- communication between external service and Open edX (Credentials);
+- implementation of service-specific badge types (templates);
+- earned badges propagation to service's user profiles;
+- users notification;
+- etc.
+
+.. note::
+    Currently, there is a single integration implemented for the `Credly (by Pearson)`_.
+
+----
+
+Credly backend
+--------------
+
+- `Overview`_
+- `Configuration`_
+- `Badge templates management`_
+- `Requirements setup`_
+- `Awarded badges`_
+- `Side effects`_
+- `More details`_
+
+Overview
+~~~~~~~~
+
+Credly badges distribution backend is implemented as a baked-in plugin.
```

Comment:
When I read plugin, I think "external requirement/library optionally installed". Just confirming, this is not that -- this functionality will be part of the base Credentials service, but is totally optional if the Open edX Operator wants to take advantage of it.

Reply:
Also -- is the `Credly` functionality optional from the rest of the Badges implementation? Or will it require the Operator to have a Credly account for any Badge functionality?

Reply:
On one hand, we want Credly extra functions to be optional, but still a part of the standard codebase (disabled by default, but available if needed out of the box). Currently, it is designed as a separate installable credentials application (which is prepared to be extracted into a separate repo, but is baked-in within its parental `badges` app and pre-added to requirements). 
This is a matter of discussion. 
The idea behind such "complexity":
- to provide a live example of an extension for the `badges` app;
- to have the ability to extract (if needed in the future) from the main repo;
- design badges extensions with respect to the core Badges configuration (this is in progress);